velox/exec/fuzzer/ReferenceQueryRunner.h

```diff
@@ -35,6 +35,23 @@ class ReferenceQueryRunner {
       const std::string& sql,
       const std::vector<RowVectorPtr>& input,
       const RowTypePtr& resultType) = 0;
+
+  /// Returns true if the underlying Query Runner supports
```

Comment:
> underlying Query Runner

The word "underlying" suggests a composition pattern where this thing contains another (underlying) thing. This is not the case here.

A less confusing comment could be "Returns 'true' if 'executeVector' is supported".

Also, 'supportsVeloxVectors' name is confusing. How come a query runner may not support Velox vectors if there are existing methods that use these (see existing 'execute' method).

Perhaps, a clearer name would specify that Vectors are support as results of a query. Something like 'supportsVeloxVectorResults' may work better.